game_eggs/steamcmd_servers/arma/arma3/egg-arma3.json

```diff
@@ -138,6 +138,15 @@
             "user_editable": true,
             "rules": "nullable|string"
         },
+        {
+            "name": "Optional Mods",
```

Comment:
Maybe change name to "Optional Client-Side Mods" to be more clear that only client-side mods would be functional in this list. Also, maybe consider mentioning in the description that (a.) this setting is only relevant when `verifySignatures = 2`, and (b.) keys must manually be deleted if a mod is removed from the list (unless the entrypoint in the image is improved, as mentioned on its PR). I like to keep descriptions as short as possible, so changing this to an "[Advanced]" variable (like the "Server-Side Only Mods" one) could allow for a less detailed description (leaving out the warnings I previously mentioned) and assume the advanced user knows what this is doing if they chose to use it.

Reply:
CHanges to the title and description have been made.